lib/run/index.js

```diff
@@ -158,19 +158,23 @@ module.exports = function (options, callback) {
                         summary: emitter.summary
                     });
 
-                    _.forEach(['environment', 'globals', 'collection'], function (item) {
+                    var ENV_EXPORT_KEYS = ['id', 'name', 'synced', 'syncedFilename'];
+
+                    _.forEach(['environment', 'globals'], function (item) {
                         // fetch the path name from options if one is provided
-                        var path = _.get(options, _.camelCase(`export-${item}`));
+                        var path = _.get(options, _.camelCase(`export-${item}`)),
+                            exportContent = emitter.summary[item].toJSON();
 
                         // if the options have an export path, then add the item to export queue
                         path && emitter.exports.push({
                             name: item,
                             default: `newman-${item}.json`,
                             path: path,
-                            content: _.merge(emitter.summary[item].toJSON(), {
+                            content: item === 'environment' ? _.merge(exportContent, {
+                                timestamp: Date.now(),
                                 _postman_exported_at: (new Date()).toISOString(),
                                 _postman_exported_using: util.userAgent
-                            })
+                            }, _.pick(_.get(options, item), ENV_EXPORT_KEYS)) : _.get(exportContent, 'values')
```

Comment:
This is NOT how it should be exported. The toJSON function should be returning consumable export data directly.